sasl_bind.c

```diff
@@ -0,0 +1,21 @@
+#include <ldap.h>
+#include "sasl_bind.h"
+
+static int
+sasl_trivial_interact (LDAP * ld, unsigned flags, void *defaults,
+                       void *sasl_interact)
+{
+  (void) ld;
+  (void) flags;
+  (void) defaults;
+  (void) sasl_interact;
+  return LDAP_SUCCESS;
+}
+
+int
+trivial_external_sasl_bind (LDAP * ld)
+{
+  return ldap_sasl_interactive_bind_s (ld, NULL, "EXTERNAL", NULL, NULL,
```

Comment:
Guys, believe me or check out openldap & cyrus-sasl sources :)

ldapmodify uses `ldap_sasl_interactive_bind_s()` with non-trivial callback and it just returns on `SASL_CB_LIST_END`.

![xxx](https://cloud.githubusercontent.com/assets/131844/21735883/5d081588-d486-11e6-9250-1522f47dcd55.png)


@wferi your callback function is good because cyrus-sasl sets required prompts to [SASL_CB_LIST_END], and all we have to do is to return.